exasol_script_languages_container_ci_setup/doctor.py

```diff
@@ -0,0 +1,91 @@
+"""
+The doctor module provides functionality to check the health of the `slc-ci-setup`
+package and also provide help to find potential fixes.
+"""
+import shlex
+import subprocess
+import sys
+from enum import Enum
+from typing import Iterator
+
+from exasol_error_reporting_python.exa_error import ExaError
+
+
+SUPPORTED_PLATFORMS = ["linux", "darwin"]
+
+
+def check_shell_cmd(cmd: str) -> bool:
+    """
+    Runs a shell command and returns True if exit code was 0, False otherwise 
+    :param cmd: shell command to execute
+    :return: returns True if exit code was 0, False otherwise
+    """
+    ret_val = True
+    completed_process = subprocess.run(shlex.split(cmd), stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    try:
+        completed_process.check_returncode()
+    except subprocess.CalledProcessError as e:
+        ret_val = False
+    return ret_val
+
+
+class ErrorCodes(Enum):
+    """The equivalent of ICD-10 codes this doctor is using"""
+
+    Unknown = ExaError.message_builder('E-SLCCS-01').message("Unknown issue")
+    TargetPlatformNotSupported = ExaError.message_builder('E-SLCCS-02')\
```

Comment:
Usually, the error codes should be defined in place

Reply:
Done